plugins/sshd.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sshd
+spec:
+  version: "v0.1.0"
+  platforms:
+  - uri: https://github.com/ottoyiu/kubectl-sshd/archive/v0.1.0.tar.gz
+    sha256: 592dbf172f4bd35317b6632e120056c803e29c5713a28d876b69baf2f9955dba
+    bin: kubectl-sshd
+    files:
+    - from: "./*/bin/static-dropbear"
+      to: .
+    - from: "./*/bin/kubectl-sshd"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin requires that the 'tar', 'ls', 'ln' binaries are present in your container. Otherwise, this plugin will fail.
```

Comment:
I think it's safe to assume that a container with `tar` also has coreutils. So consider to drop the extra mention of `ls` and `ln`. Also drop the second sentence. It just restates that this is a requirement.